task/04-arrays-tasks.js

```diff
@@ -415,7 +450,10 @@ function toStringList(arr) {
  *      { country: 'Russia',  city: 'Saint Petersburg' }
  */
 function sortCitiesArray(arr) {
```

Comment:
please fix block (i mean if and else if blocks) formatting according to google code style https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks